### DISCUSSION TITLE
Does changelog fetching only support GitHub and GitLab platforms?

### DISCUSSION THREAD

**Discussion by AndreasPetersen** [renovatebot/renovate#28070] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

bitbucker-server

### Please tell us more about your question or problem

I'm hosting a Helm chart on our private `bitbucket-server` platform. I've added sources to the `Chart.yaml`, and have a `CHANGELOG.md`. When Renovate updates it writes:

```
INFO: Unknown platform, skipping changelog fetching. (repository=PROJECT/my-service, baseBranch=master, branch=renovate/master-microservice-2.x)
     "sourceUrl": "https://my-host.com/projects/PROJECT/repos/microservice-helm-chart",
     "hostType": null
```

According to the Renovate log:

```
Adding password authentication for my-host.com (hostType=bitbucket-server) to hostRules
```

I have also tried adding to my config:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "local>PROJECT/renovate",
  ],
  "hostRules": [
    {
      "matchHost": "my-host.com",
      "hostType": "bitbucket-server"
    }
  ],
  "packageRules": [
    {
      "matchDatasources": ["helm"],
      "hostType": "bitbucket-server"
    }
  ]
}
```

Yet I still get the same error. Looking at 

https://github.com/renovatebot/renovate/blob/fb9303c19081e341c123581c00b3e72701c1c8c9/lib/workers/repository/update/pr/changelog/index.ts#L31-L47

and 

https://github.com/renovatebot/renovate/blob/fb9303c19081e341c123581c00b3e72701c1c8c9/lib/modules/platform/util.ts#L14

it looks like only the platforms `github` and `gitlab` are supported. Is this correct? I haven't been able to find any documentation stating this possible limitation.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Correct

**Reply by AndreasPetersen** [reply]
Thanks for the quick reply!

**Comment by AndreasPetersen** [comment]
I also tried setting https://docs.renovatebot.com/configuration-options/#customchangelogurl, but this gives the same error, despite the fact that `customChangelogUrl` states:

> Renovate can fetch changelogs from Bitbucket, Gitea (Forgejo), GitHub and GitLab platforms only, and setting the URL to an unsupported host/platform type won't change that.

It's a bit unclear if "Bitbucket" includes both server and cloud variants.

**Reply by rarkins** [reply]
In Renovate Bitbucket usually refers to Bitbucket Cloud, and we use "Bitbucket Server" (still) for BBS and BBDC
